File: server/src/main/kotlin/com/pidulgi/server/admin/dto/response/AdminGetMemberDetailResponse.kt
package com.pidulgi.server.admin.dto.response

import com.pidulgi.server.member.dto.response.MemberImageResponse
import com.pidulgi.server.member.entity.type.Gender
import com.pidulgi.server.point.dto.response.PointTransactionResponse
import java.time.LocalDateTime

data class AdminGetMemberDetailResponse(

    val memberId: Long,
    val uuid: String,
    val phoneNumber: String,
    val nickname: String,
    val gender: Gender,
    val birthYear: Int,
    val comment: String?,
    val bio: String?,
    val createdAt: LocalDateTime,
    val updatedAt: LocalDateTime,

    // 이미지
    val publicImages: List<MemberImageResponse>,
    val privateImages: List<MemberImageResponse>,

    // 포인트 내역
    val pointTransactions: List<PointTransactionResponse>,
)

File: admin/types/AdminGetMemberDetailResponse.ts
import { AdminGetMemberImageResponse } from "./AdminGetMemberImageResponse";

export type AdminGetMemberDetailResponse = {
  memberId: number;
  uuid: string;
  phoneNumber: string;
  nickname: string;
  birthYear: number;
  gender: "MALE" | "FEMALE";
  bio: string | null;
  comment: string | null;
  createdAt: string;
  updatedAt: string;
  deletedAt: string | null;
  images: AdminGetMemberImageResponse[];
};
